tests/ui/imports/glob-conflict-cross-crate-2.rs

```diff
@@ -0,0 +1,12 @@
+// check-pass
+// aux-build:glob-conflict-cross-crate-2-extern.rs
+
+extern crate glob_conflict_cross_crate_2_extern;
+
+use glob_conflict_cross_crate_2_extern::*;
+
+fn main() {
+    let _a: C = 1;
```

Comment:
Breaking change: code that previously failed to compile will now compile successfully, but with a warning.